assets/css/style.css

```diff
@@ -209,6 +219,8 @@ nav {
   color: #000000;
   font-family: "Open Sans", sans-serif;
   font-weight: 700;
+  margin-top: 0;

```

Comment:
If you don't want to use origin margin value you can define it at the top

*{margin:0;}